jvm/experiment.yaml

```diff
@@ -27,18 +27,14 @@ spec:
       lowerParameter: heap_size
       upperParameter: memory
   metrics:
-  - name: duration
-    minimize: true
-    query: "{{duration .StartTime .CompletionTime}}"
-  - name: cost
-    minimize: true
-    query: "{{div (add (mul .Values.cpu 22) (mul .Values.memory 3)) 1000}}"
+  - minimize: true
+    name: duration
+    query: '{{duration .StartTime .CompletionTime}}'
+  - minimize: true
+    name: cost
+    query: '{{div (add (mul .Values.cpu 22) (mul .Values.memory 3)) 1000}}'
```

Comment:
Most of these changes just look like alphabetical field sorting which doesn't make as much sense (I assume this is part of the current conversion process), can you put the fields back in the previous order for readability?

Reply:
Sure, good catch, thanks